src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -2015,8 +2028,21 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Output TaskParameter="FilesWritten" ItemName="FileWrites"/>
       <Output TaskParameter="DependsOnSystemRuntime" PropertyName="DependsOnSystemRuntime"/>
     </ResolveAssemblyReference>
+
+    <ItemGroup>
+      <ReferencePathWithInterfaceOnlyAssemblies Include="@(ReferencePath->'%(ReferenceAssembly)')" />
```

Comment:
Ah, the item def group is the next diff, ignore :)

---
In reply to: [114407016](https://github.com/Microsoft/msbuild/pull/2039#discussion_r114407016) [](ancestors = 114407016)